project/CelebornBuild.scala

```diff
@@ -750,8 +756,9 @@ object CelebornMaster {
         Dependencies.ratisServer,
         Dependencies.ratisShell,
         Dependencies.scalatestMockito % "test",
-      ) ++ commonUnitTestDependencies ++ mpuDependencies
+      ) ++ commonUnitTestDependencies ++ mpuDependencies ++ jmhDependencies
     )
+    .enablePlugins(pl.project13.scala.sbt.JmhPlugin)
```

Comment:
do we have to enable this plugin with sbt?

The jmh  dependencies are involved by this plugin.